go.mod

```diff
@@ -31,6 +31,7 @@ require (
 	github.com/prometheus/common v0.9.1
 	github.com/prometheus/prometheus v0.0.0-00010101000000-000000000000
 	github.com/shurcooL/httpfs v0.0.0-20190707220628-8d4bc4ba7749
+	github.com/shurcooL/vfsgen v0.0.0-20181202132449-6a9ea43bcacd // indirect
```

Comment:
That line is usually gone after `go mod tidy`

Reply:
I have git hook on make check which runs go mod tidy. Not sure what is going on here :/